x-pack/plugins/security_solution/public/management/common/routing.ts

```diff
@@ -327,3 +349,31 @@ export const getEventFiltersListPath = (location?: Partial<EventFiltersPageLocat
     querystring.stringify(normalizeEventFiltersPageLocation(location))
   )}`;
 };
+
```

Comment:
I'm thinking at some point we should split this file in subfiles as it's getting bigger and bigger.

Reply:
Yes we should, and also we should refactor it, there's a lot of repeating code, specially since each page is keeping track of its own location state.